documentation/src/docs/deployment/run-local.md

```diff
@@ -132,7 +139,6 @@ Caused by: sun.security.validator.ValidatorException: PKIX path building failed:
                  .validateHostName(false)
                  .build();
     ```
-    Everything else should be the same as other reader/writer apps.
-
+6. Everything else should be the same as the Reader/Writer apps.
 
-6. Clients can then connect to the controller at `${HOST_IP}:9090`.
+These properties include ports for Zookeeper, Segment Store and Controller. They also contain other configurations related to security.
```

Comment:
Do we need this statement here? I don't think it is relevant for the user. What do you say?

Reply:
Accepted sugesstion